cmd/influx_inspect/buildtsi/buildtsi.go

```diff
@@ -416,3 +424,20 @@ func isRoot() bool {
 	user, _ := user.Current()
 	return user != nil && user.Username == "root"
 }
+
+func modelsFieldType(block byte) models.FieldType {
+	switch block {
+	case tsm1.BlockFloat64:
+		return models.Float
+	case tsm1.BlockInteger:
+		return models.Integer
+	case tsm1.BlockBoolean:
+		return models.Boolean
+	case tsm1.BlockString:
+		return models.String
+	case tsm1.BlockUnsigned:
+		return models.Unsigned
+	default:
+		return models.Empty
+	}
+}
```

Comment:
Can this be moved to `tsm1` since it's duplicated in the `importer` package as well?

Reply:
As it turns out, they aren't really duplicated. One operates on the encoded byte in the tsm block, and the other operates on some value out of a protobuf. These values don't match.